src/subcommand/wallet/inscribe.rs

```diff
@@ -148,6 +148,33 @@ impl Inscribe {
 
         sat = batchfile.sat;
       }
+      (None, None, Some(delegate)) => {
+        parent_info = wallet.get_parent_info(self.parent, &utxos)?;
+
+        postage = self.postage.unwrap_or(TARGET_POSTAGE);
+
+        ensure! {
+          wallet.inscription_exists(delegate)?,
+          "delegate {delegate} does not exist"
+        }
```

Comment:
Let's add a flag called `--allow-non-existant-delegate` and then skip this check if it is set.

Reply:
what is the expected response for non-existent-delegate w/o payload and existent-delegate w/ payload?